spec/support/seeds/seeds_e2e.rb

```diff
@@ -56,9 +56,6 @@
   user.add_role(:super_admin)
   user.add_role(:single_resource_admin, Config)
   user.add_role(:trusted)
-
-  # Enable new Admin Members
-  FeatureFlag.enable(:new_admin_members, user)
```

Comment:
The old feature flag we are no longer using.

Reply:
I already have this covered in the [PR](https://github.com/forem/forem/pull/16475) that removes all the previous Admin Member code. 😊

Reply:
Awesome. I'll update this branch with main, one that PR is merged. 😎